app/views/pages/about.html.erb

```diff
@@ -4,56 +4,63 @@
+                    SORNs that use the <a class="usa-link usa-link--external" href="https://www.whitehouse.gov/sites/whitehouse.gov/files/omb/circulars/A108/omb_circular_a-108.pdf">OMB A-108(r.2016) templates</a> are much easier to parse into structured records. As we continue developing this dashboard, we’ll add methods to parse documents that will capture the content from a larger share of the total number of documents. SORNs that are not parsed, or partially parsed, are still included in search results, but only if the search is of the full text of the SORN, not in a search of specific sections.
                     </p>
 
                     <p>
                     The dashboard will add and parse all new SORNs published using any of the five A-108 templates.
                     </p>
 
-                    <h3 id="section-heading-h3">How do I download my results as a CSV file?</h3>
+                    <h3 class="about__h3">Saving results</h3>
                     <p>
                     Clicking <strong>Download results as a CSV file</strong> prepares a comma-separated variable file containing rows for the search results on screen, and columns for each SORN section that is selected in the sidebar. You can add or remove columns from the output file by selecting or deselecting items from the sidebar. The output file contains the full text for all selected columns. If you want to download the complete structured content of every SORN in our database, reload the page and check all SORN sections in the sidebar and then click the save as csv button (note: this file will be very large!)
                     </p>
 
-                    <h2 id="section-heading-h2">Feedback</h2>
+                    <h2 class="about__h2">Feedback</h2>
                     <p>
-                    We are still actively developing this prototype and we want to hear your ideas and feedback. If you are interested in participating in user research or want to get in contact with us for any other reason, please reach out to us at: <a href="mailto:privacy_devops@gsa.gov">privacy_devops@gsa.gov</a>
+                    We are still actively developing this prototype and we want to hear your ideas and feedback. If you are interested in participating in user research or want to get in contact with us for any other reason, please reach out to us at: <a class="usa-link usa-link--external" href="mailto:privacy_devops@gsa.gov">privacy_devops@gsa.gov</a>
```

Comment:
@igorkorenfeld the address i gave gsa was sorn-dash, but since we don't have that yet, i wonder if we want to hide this?